rpm/podman.spec

```diff
@@ -187,7 +189,16 @@ when `%{_bindir}/%{name}sh` is set as a login shell or set as os.Args[0].
 Summary: Metapackage for setting up %{name} machine
 Requires: %{name} = %{epoch}:%{version}-%{release}
 Requires: gvisor-tap-vsock
-Requires: qemu
+%if %{defined qemu}
+%ifarch aarch64
+Requires: qemu-system-aarch64-core
+%else
+Requires: qemu-system-x86-core
+%endif
+%else
+Requires: qemu-kvm
+%endif
+Requires: qemu-img
```

Comment:
Not a biggie but will podman-machine have some use of qemu-img (just for handling image formats) without the actual qemu - only with qemu-kvm?

Reply:
`qemu-img` will be required to resize the qcow2 image on podman machine init or podman machine set AFAIK.
But without the actual qemu-kvm/qemu-system there is no way to actually run/start the VM so it would be quite useless without them.

Reply:
i'm going as per https://github.com/containers/podman/pull/24369#discussion_r1883866655